core/src/main/scala/org/apache/spark/rdd/NewHadoopRDD.scala

```diff
@@ -195,6 +205,10 @@ class NewHadoopRDD[K, V](
               e)
             finished = true
             null
+          case e: FileNotFoundException if ignoreMissingFiles =>
```

Comment:
shall we also apply https://github.com/apache/spark/pull/19868/files#diff-83eb37f7b0ebed3c14ccb7bff0d577c2R273 here?